### DISCUSSION TITLE
Recolorable ties on the hybrid coaster

### DISCUSSION THREAD

**Discussion by fidwell** [OpenRCT2/OpenRCT2#19943] [Feature requests] [answered]
On the hybrid coaster, the track ties (sorry if that's not the right term, hopefully you know what I mean) are always gray and cannot be recolored. However, with [Wildcat's Revenge](https://rcdb.com/20462.htm), there is now precedent for black ties, so it would be nice to be able to recolor these.

**Comment by X123M3-256** [comment, marked as answer]
There was already an example with painted ties when the track was under development - Hakugei in Japan, so this was considered and I decided against doing it at the time.

The hybrid coaster track already uses both its remap colors so this would require support for track sprites with three remap colors as does the other idea, which was to use the third remap to allow differently colored rails like Joker has. I prefer this idea over remappable ties but I decided against that too in the end.

You also need to consider the loss of color resolution - there are more shades of gray in the palette than the 12 remappable ones, and so if you make remappable ties and paint them gray, they almost certainly won't be identical to the current sprites. If they end up looking worse I don't think it's worth it, especially as getting the ties to look good enough was difficult as it was (and there were some angles I was never really happy with).

**Reply by fidwell** [reply]
That's logical. Thanks for the explanation.
